README.md

```diff
@@ -64,14 +44,19 @@ For general use, use the `main` branch along with [version 2.15.4 of the CodeQL
         use 'codeql resolve qlpacks' and 'codeql resolve languages'.
     ```
 
-1. Install CodeQL Packages using `codeql pack install`
+1. Install CodeQL Packages 
+    
+    For WHCP_21H2 and WHCP_22H2 branches:
+ 
+    **NOTE** Visual Studio 17.8 broke compatibility with the older versions of CodeQL used in the WHCP_21H2 and WHCP_22H2 branches. CodeQL 2.15.4 has been validated for use with WHCP 21H2 and WHCP 22H2. If using Visual Studio version 17.8+, follow the steps for ALL OTHER BRANCHES, below. **Make sure to remove the CodeQL submodule if it exists in your repo.** CodeQL might try to use the queries in the submodule by default which will cause errors because of mismatched versions.
 
-   For WHCP BRANCHES: Skip this step.
+    The WHCP branches WHCP_21H2 and WHCP_22H2 use the CodeQL repository as a submodule instead of using CodeQL packs. When using these branches, make sure to update submodules. 
```

Comment:
This paragraph is a bit confusing, because it provides a notice about using 17.8 and CodeQL packs, then immediately instructs people to use the "ALL OTHER BRANCHES" section, and then follows that telling users to update submodules.  I think we should reword this to make it more clear that if and only if you are on VS 17.7 or below should you use submodules for these branches, and for all other cases (17.8 or other branches) you should use the pack commands.